apps/condo/domains/organization/queries/OrganizationEmployee.graphql

```diff
@@ -0,0 +1,95 @@
+query getActiveOrganizationEmployee($where: OrganizationEmployeeWhereInput!) {
```

Comment:
Maybe we can make this query more precise?

Like to move `isBlocked` and `type` here, only left `id` here

Reply:
I want to filter by userId right away, so I left id and userId